MaxMind.Db/Reader.cs

```diff
@@ -247,7 +252,10 @@ private int StartNode(int bitLength)
         private T Deserialize<T>(JToken value)
         {
             var serializer = new JsonSerializer();
-            return serializer.Deserialize<T>(new JTokenReader(value));
+            using (var reader = new JTokenReader(value))
+            {
+                return serializer.Deserialize<T>(reader);
+            }
         }
```

Comment:
No one had reported an issue. I haven't dug through the `JTokenReader` source to see why it is `IDisposable`.

I just pushed a release notes update that mentions it.